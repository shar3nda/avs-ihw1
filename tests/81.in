10
9 2 9 10 9 6 10 -1 2 0