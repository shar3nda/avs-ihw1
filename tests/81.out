-3 -10 -8 4 7 10 -3 1 9